selfdrive/locationd/locationd.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <fstream>
+#include <string>
+#include <memory>
+
+#include <eigen3/Eigen/Dense>
+
+#include "messaging.hpp"
+#include "common/params.h"
+#include "common/util.h"
+#include "common/swaglog.h"
+#include "common/timing.h"
+#include "common/transformations/coordinates.hpp"
+#include "common/transformations/orientation.hpp"
+
+#include "models/live_kf.h"
+
+#define DEG2RAD(x) ((x) * M_PI / 180.0)
+#define RAD2DEG(x) ((x) * 180.0 / M_PI)
```

Comment:
these should probably go in a generic header, maybe selfdrive/common/util.h?

Reply:
also defined in sensord/sensors/bmx055_gyro.cc

Reply:
put it in common/transformations/coordinates.hpp now, where the .cc was already using it